gr-qtgui/lib/QFosphorColorMapper.cc

```diff
@@ -64,10 +67,13 @@ int QFosphorColorMapper::loadFromFile(QFile& file)
 
         /* Are we in a palette ? */
         if (kp < 0) {
-            sscanf(line.constData(), "%d %ms", &kp, &name);
+            ss.str() = line.toStdString();
```

Comment:
I found this needs to be ss.str(line.toStdString()); instead, it looks like ss.str() just returns a copy of the current contents rather than a reference that can be used to set the contents of the stringstream.